docs/timeouts.md

```diff
@@ -17,48 +17,49 @@ be loaded and the new URL to appear before continuing.
 
  - How to change: To change globally, add `getPageTimeout: timeout_in_millis` to your Protractor configuration file. For an individual call to `get`, pass an additional parameter: `browser.get(address, timeout_in_millis)`
 
-**Waiting for Page Synchronization**
+### Waiting for Angular
 
-Before performing any action, Protractor asks Angular to wait until the page is synchronized. This means that all timeouts and http requests are finished. If your application continuously polls $timeout or $http, it will
-never be registered as completely loaded. You should use the
-$interval service ([interval.js](https://github.com/angular/angular.js/blob/master/src/ng/interval.js)) for anything that polls continuously (introduced in Angular 1.2rc3).
+Before performing any action, Protractor waits until there are no pending asynchronous tasks in your Angular application. This means that all timeouts and http requests are finished. If your application continuously polls $timeout or $http, Protractor will wait indefinitely and time out. You should use the
+[$interval](https://github.com/angular/angular.js/blob/master/src/ng/interval.js) for anything that polls continuously (introduced in Angular 1.2rc3).
 
- - Looks like: an error in your test results - `Timed out waiting for Protractor to synchronize with the page after 11 seconds.`
+ - Looks like: an error in your test results - `Timed out waiting for asynchronous Angular tasks to finish after 11 seconds.`
 
  - Default timeout: 11 seconds
 
  - How to change: Add `allScriptsTimeout: timeout_in_millis` to your Protractor configuration file.
```

Comment:
Let's add a 'You can also disable waiting for Angular, see below' (and have a link to that header)